src/Umbraco.Tests.UnitTests/Umbraco.Infrastructure/Migrations/PostMigrationTests.cs

```diff
@@ -10,19 +10,19 @@
 using Umbraco.Core.Persistence.SqlSyntax;
 using Umbraco.Core.Scoping;
 using Umbraco.Core.Services;
-using Umbraco.Persistance.SqlCe;
 using Umbraco.Tests.Testing;
 
-namespace Umbraco.Tests.Migrations
+namespace Umbraco.Tests.UnitTests.Umbraco.Infrastructure.Migrations
 {
     [TestFixture]
     public class PostMigrationTests
     {
-        private static ILoggerFactory _loggerFactory = NullLoggerFactory.Instance;
+        private static readonly ILoggerFactory s_loggerFactory = NullLoggerFactory.Instance;
```

Comment:
Minor thing - is this rename unintentional?

Reply:
Yes, it is part of the new linting rules, because it is static